matter/wifi/rs911x/hal/efx_spi_intf.c

```diff
@@ -63,6 +63,16 @@ static void dma_init(void)
 /****************************************************************************
  * Initialize SPI peripheral
  *****************************************************************************/
+void rsi_sem_init(void)
+{
+  spi_sem = xSemaphoreCreateBinary();
+  xSemaphoreGive(spi_sem);
+  dma_init();
```

Comment:
This shouldn't be there. it's a semaphore init function, not a dma or peripherical init function

Reply:
DMA Init should be elsewhere with all of the other peripheral initializations